server/channels/app/upload.go

```diff
@@ -271,6 +271,7 @@ func (a *App) UploadData(c request.CTX, us *model.UploadSession, rd io.Reader) (
 	}
 
 	info.CreatorId = us.UserId
+	info.ChannelId = us.ChannelId
```

Comment:
I am pretty surprised we didn't assign this value before. Have you found if we assign it somewhere else or if there is any strong reason not to assign it?

Reply:
@larkox I think the reason is merely that at the time of implementation we didn't have a `FileInfo.ChannelID`. The mapping and permissions would happen indirectly through any attached post ids.

Reply:
As long as we don't think that this will break anything, this probably should be assigned here. I found it very odd that it was missing in the first place. I'm not sure where exact it was assigned elsewhere, but since we do provide the channel scope now, we should assign it here.